frontend/device/remarkable/device.lua

```diff
@@ -153,8 +150,48 @@ function Remarkable:init()
     local scalex = screen_width / self.mt_width
     local scaley = screen_height / self.mt_height
 
-    self.input:registerEventAdjustHook(adjustAbsEvt)
-    self.input:registerEventAdjustHook(self.adjustTouchEvent, {mt_scale_x=scalex, mt_scale_y=scaley})
+    -- Assume input stuff is saner on mainline kernels...
+    -- (c.f., https://github.com/koreader/koreader/issues/10012)
+    local is_mainline = false
+    --- @fixme: Find a better way to discriminate mainline from stock...
+    local std_out = io.popen("uname -r", "r")
+    if std_out then
+        local release = std_out:read("*line")
+        std_out:close()
+        release = release:match("^(%d+%.%d+)%.%d+.*$")
+        release = tonumber(release)
+        if release and release >= 6.2 then
+            is_mainline = true
+        end
+    end
+    if is_mainline then
+        local mt_height = self.mt_height
+        local mainlineInputMangling = function(this, ev)
+            if ev.type == C.EV_ABS then
+                -- Mirror Y for the touch panel
+                if ev.code == C.ABS_MT_POSITION_Y then
```

Comment:
This needs to be `if ev.code == C.ABS_Y then`

Reply:
That would break Wacom, which is the whole cnundrum ;o).